README.md

```diff
@@ -1,42 +1,69 @@
+- Flexible
+    - Supports many different input types such as file, journald, windows events, tcp / udp, and external APIs (cloudwatch, azure log analytics) as well as parsing with json and regex.
+    - Easily extended by writing an "operator" or "plugin" which is just a unit of code that performs a task such as reading data from a source, parsing data, or shipping data.
+- Lightweight with low resource consumption
+    - Uses next to no resource while idling. It does not pollute the system with tons of clutter, it exists strictly in /opt/observiq/stanza with just a few files.
+- Written in Go
+    - Everything is self contained into a single binary / file. There are no external dependency libraries that need to exist on the system, unlike dynamically linked applications (C, C++, C#, Java, Python, Ruby, NodeJS)
+- High Performance
+    - Stanza is proven to be significantly faster than fluentd and more stable at high throughput than fluentbit.
 
+## Supported Plugins
 
-# Quick Start
+Utilize Plugins to get up and running quickly. These are some of the different plugins supported by Stanza, with more being developed all the time.
 
-Want to get started right away? Check out our [Quick Start Guide](./docs/README.md)!
+<p align="center"><img src="docs/images/stanza_plugins.png?raw=true"></p>
 
+# Documentation
 
-## Community
+## [Quick Start](./docs/README.md)
 
-Stanza is an open source project. If you'd like to contribute, take a look at our [contribution guidelines](./CONTRIBUTING.md) and [developer guide](./docs/development.md). We look forward to building with you.
+### Installation
 
+To install Stanza, we recommend using our single-line installer provided with each release. Stanza will automatically be running as a service upon completion. 
 
-## Roadmap
+#### Linux/macOS
+```shell
+sh -c "$(curl -fsSlL https://github.com/observiq/stanza/releases/latest/download/unix-install.sh)" unix-install.sh
+```
+#### Windows
+```pwsh
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; Invoke-Expression ((New-Object net.webclient).DownloadString('https://github.com/observiq/stanza/releases/latest/download/windows-install.ps1')); Log-Agent-Install
+```
 
-Stanza uses GitHub [projects](https://github.com/observIQ/stanza/projects) to organize future work. There's a lot to do, and more use cases are identified almost daily. We'll keep it organized and moving forward the best we can, but we're always looking for great ideas and contributions of all sizes.
+For more details on installation and configuration, check out our full [Quick Start Guide](./docs/README.md)!
 
+# Community
+
+Stanza is an open source project. If you'd like to contribute, take a look at our [contribution guidelines](./CONTRIBUTING.md) and [developer guide](./docs/development.md). We look forward to building with you.
```

Comment:
While this is true, we have not settled on a real contribution process utlizing [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement). Im wondering if we should omit this section if we do not actually have a contribution process.

Reply:
We'll keep this for now as it highlights Stanza being open-source